### PR TITLE
Sort-string, pick, omit tasks solution

### PR DESCRIPTION
**Pull request by baidauletov**
### Sort-strings
    String function _localeCompare_ use to sort 'strings' array 

### Pick
    _Object.entries_ and Array function _filter_ use to pick items by 'fields' list from 'obj'

### Omit 
    Same as **Pick** task,  _Object.entries_ and Array function _filter_ use, 
    but now for omit items by 'fields' list from 'obj'

### CODE REVIEW

**Comment by jsru** [comment]
Добавляю преподавателя (@dosandk) для код-ревью.

File: 02-javascript-data-types/2-pick/index.js [hunk 1]
```diff
@@ -5,5 +5,9 @@
  * @returns {object} - returns the new object
  */
 export const pick = (obj, ...fields) => {
+  let entries = Object.entries(obj);
```
Comment (dosandk) [inline 02-javascript-data-types/2-pick/index.js:8]: `const`

File: 02-javascript-data-types/3-omit/index.js [hunk 2]
```diff
@@ -5,5 +5,9 @@
  * @returns {object} - returns the new object
  */
 export const omit = (obj, ...fields) => {
+  let entries = Object.entries(obj);
```
Comment (dosandk) [inline 02-javascript-data-types/3-omit/index.js:8]: `const`

**Review by dosandk — APPROVED** [review]
good 👍

**Pull request merged**